### PR TITLE
Refactor initPodcastPlayback

### PR DESCRIPTION
**Pull request by netochaves**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Documentation Update

## Description
Fix eslint errors in `initializePodcastPlayback.js`
## Related Tickets & Documents
#3777 
## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
N/A
## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

### CODE REVIEW

File: app/assets/javascripts/initializers/initializePodcastPlayback.js [hunk 1]
```diff
@@ -1,3 +1,7 @@
+'use strict';
+
+/* eslint-disable no-param-reassign */
+/* global time */
 /**
  * This script hunts for podcast's "Record" for both the podcast_episde's
```
Comment (rhymes) [inline app/assets/javascripts/initializers/initializePodcastPlayback.js]: there's a typo here, `podcast_episde` instead of `podcast_episode`

**Comment by rhymes** [comment]
Hi @netochaves, thanks again! 

I think you need to update your local copy of master, I noticed in this and a previous PR how all other files in your PRs are not in sync.

Please cut branches from a fresh master, to keep in sync you can regularly issue a 

`git checkout master && git pull upstream master`

For this branch after pulling from master you can issue a `git rebase master` and then `git push -f` to update the remote PR

**Comment by netochaves** [comment]
@rhymes I rebase with master and made some changes.

File: app/assets/javascripts/initializers/initializePodcastPlayback.js [hunk 2]
```diff
@@ -1,5 +1,9 @@
+'use strict';
+
+/* eslint-disable no-param-reassign */
```
Comment (rhymes) [inline app/assets/javascripts/initializers/initializePodcastPlayback.js]: this should probably be disabled locally with /* eslint-disable-next-line no-param-reassing */

**Comment by mstruve** [comment]
Hey @netochaves! 

As I stated in your other PR #4355  we are in the process of going through and cleaning up some of our PR history to help us better prioritize what we need to be focusing on. Since this PR has not been active for a while and has some conflicts I am going to close it for now. If you would still like to get this merged please feel free to resolve the conflicts and then open the PR back up. We would be more than happy to help push this through if you would like!

Thanks so much for being an awesome contributor! Ping us anytime if you have questions!

**Pull request closed without merging**